pims/bioformats.py

```diff
@@ -5,32 +5,53 @@
 from pims.base_frames import FramesSequence
 from pims.frame import Frame
 from warnings import warn
-from os.path import isfile
+import os
 
 try:
-    import javabridge
+    import jpype
 except ImportError:
-    javabridge = None
+    jpype = None
 
-try:
-    import bioformats
-except ImportError:
-    bioformats = None
+LOCI_TOOLS_PATH = os.path.join(os.path.dirname(__file__),
```

Comment:
I don't really like these search cascades and for simplicity would prefer putting it inside the pims folder. This way it seems to get pretty complicated.

What do you think of downloading it as part of `setup.py` (if jpype is there), or else suggest the user to download it to pims folder by himself?